### ISSUE TITLE
v1.15.8: All illumination values lost

### ISSUE THREAD

**Issue by juersei** [ioBroker/ioBroker.hm-rpc#508]
After updatig to v1.15.8 I lost all illumination values (being reported to be 0 in SQL)  from two different HmIP devices:

movement and light sensor 
weather station

Interesting: All other parameter from weather station are working flawlessly (wind, rain, temperature, ....)

**Comment by foxriver76** [comment]
Can you show the exact names of the now missing values. There seems to be something worng with 1.15.8 but until now no one has provided a reproducing example.

**Comment by foxriver76** [comment]
Unfortunately response cannot correctly handle the images, thus they seem to be missing.

**Comment by juersei** [comment]
Hi Max, 

here are the screenshots from weather station and movement detector:

![grafik](https://user-images.githubusercontent.com/95508833/158811821-f0ecb7e7-6953-477b-9f0f-918d3a9e2478.png)
![grafik](https://user-images.githubusercontent.com/95508833/158811869-5bc8807a-a009-4dfa-8256-d96f41dd6408.png)

**Comment by foxriver76** [comment]
Please upgrade to 1.15.9 and give feedback.

**Comment by juersei** [comment]
I regret to report some issues which I have detected right now and which might be due to the earlier v 1.15.8 already.

Right now, even after the update to v.1.15.9 the communication to the CCU seems to be severely degraded (interrupted) at least for 
the weather station HmIP-SWO-PR
movement detection sensor HmIP-SMI55
temperature/humidity sensors and control devices: HmIP-STH and HmIP-WTH2 

Surprisingly the dual temp. sensor HM-WDS30-OT2-SM  seems to work flawlessly, probably since it´s not of  kind Hm**IP**?? 

I have collected all my findings incl. the correct value strings into the atttached txt file. In case of "???" I don´t record this value in SQL, therefore I don´t have any reliable information on that one
[hm-rpc v1.15.8-9 observations for weather station and movement detector.txt](https://github.com/ioBroker/ioBroker.hm-rpc/files/8303903/hm-rpc.v1.15.8-9.observations.for.weather.station.and.movement.detector.txt)
.
